taskfiles/deps/installs/msgpack-cxx.yaml

```
@@ -0,0 +1,31 @@
+            - "-DCMAKE_INSTALL_MESSAGE=LAZY"
+            - "-DCMAKE_POLICY_DEFAULT_CMP0074=NEW"
+            - "-DMSGPACK_BUILD_DOCS=OFF"
+            - "-DMSGPACK_CXX20=ON"
+          DEPS:
+            ref: ".DEPS"
+          INSTALL_TASK_NAME: "msgpack-cxx"
+          LIB_NAME: "msgpack-cxx"
+          TARBALL_SHA256: "7504b7af7e7b9002ce529d4f941e1b7fb1fb435768780ce7da4abaac79bb156f"
+          TARBALL_URL: "https://github.com/msgpack/msgpack-c/releases/download/cpp-{{.VERSION}}\
+          /msgpack-cxx-{{.VERSION}}.tar.gz"
+
```

YAMLlint (1.37.1)

[error] too many blank lines (1 > 0)

(empty-lines)